Make.inc

```diff
@@ -652,6 +652,32 @@ ifneq (,$(filter $(ARCH), powerpc64 ppc64))
 $(error Big-endian PPC64 is not supported, to ignore this error, set ARCH=ppc64le)
 endif
 
+# File name of make binary-dist result
+ifeq ($(JULIA_BINARYDIST_FILENAME),)
+DIST_OS:=$(OS)
```

Comment:
I like that idea.  I think we can assume that `tr` is available on all these misc. systems.